lib/plugins/exchange/ccxt.js

```diff
@@ -0,0 +1,51 @@
+var ccxt = require('ccxt')
+const coinUtils = require('../../coin-utils')
+const common = require('../common/ccxt')
+const kraken = require('./kraken')
+const bitstamp = require('./bitstamp')
+const itbit = require('./itbit')
+const consts = require('./consts')
+
+const ALL_EXCHANGES = {
+  kraken,
+  bitstamp,
+  itbit
+}
+
+function trade (side, account, cryptoAtoms, fiatCode, cryptoCode, exchangeName) {
+  try {
+    const exchangeConfig = ALL_EXCHANGES[exchangeName]
+
+    if (!exchangeConfig) throw Error('no exchange')
+    if (exchangeConfig.isConfigValid && !exchangeConfig.isConfigValid(account)) throw Error('Invalid config')

```

Comment:
Maybe it would be good to do a desconstruct of exchangeConfig as well. Makes the code more readable.

```js
const { isConfigValid, loadConfig, loadOption, AMOUNT_PRECISION, ORDER_TYPE} = exchangeConfig
```

Reply:
This line can be

```js
     if (_.isFunction(isConfigValid) && !isConfigValid(account)) throw Error('Invalid config')
```